pkg/identity-manager/handlers.go

```diff
@@ -253,6 +254,26 @@ func (h *Handlers) getPolicies(params policyOperations.GetPoliciesParams, princi
 	return policyOperations.NewGetPoliciesOK().WithPayload(policyModels)
 }
 
+func (h *Handlers) getPolicy(params policyOperations.GetPolicyParams, principal interface{}) middleware.Responder {
+
+	defer trace.Tracef("get policy name '%s'", params.PolicyName)()
+	var policy Policy
+
+	name := params.PolicyName
+	if err := h.store.Get(IdentityManagerFlags.OrgID, name, entitystore.Options{}, &policy); err != nil {
```

Comment:
Can you add FilterExists() to the entititystore.Options.Filter, even though we hard delete policies I would like to future proof. Also, can you make a similar change to deletePolicy?

Reply:
fixed